.github/workflows/spring-native-image.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: graalvm/setup-graalvm@v1
         with:
-          version: ${{ matrix.version }}
+          version: '22.3.0' # temporarily locked to 22.3 for Spring Boot 3 compatibility
```

Comment:
Please keep `${{ matrix.version }}` here and use this instead: 
```
version: [22.3.0] # temporarily locked to 22.3 for Spring Boot 3 compatibility
```

Reply:
Followed in https://github.com/graalvm/graalvm-demos/pull/168/commits/5ad973dc312a856a73b47064964b9550ddfdf442